tests/distributions/test_zero_inflated.py

```diff
@@ -27,30 +29,36 @@ def test_zid_shape(gate_shape, base_shape):
 @pytest.mark.parametrize("rate", [0.1, 0.5, 0.9, 1.0, 1.1, 2.0, 10.0])
 def test_zip_0_gate(rate):
     # if gate is 0 ZIP is Poisson
-    zip_ = ZeroInflatedPoisson(torch.zeros(1), torch.tensor(rate))
+    zip1 = ZeroInflatedPoisson(torch.tensor(rate), gate=torch.zeros(1))
```

Comment:
for shapes to match this should be `gate=torch.tensor(0.)` or `gate=torch.zeros(())`

Reply:
(I guess this was also incorrect in the original version of this PR)